CHANGELOG.md

```diff
@@ -0,0 +1,58 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+    PROGRAM
```

Comment:
The changelog should be human readable bullet points. I feel it might be ok to not have any until we are at the initial release. But then it needs to be detailed.